dashboard/app/assets/javascripts/dashboard.controller.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  'use strict';
+
+  angular.module('qat-dashboard').controller('DashboardController', [
+    'testExecutor',
+    function(testExecutor) {
+      var vm = this;
```

Comment:
@mDibyo I think that should be fine. With Angular we usually don't use that pattern.